test/org/zaproxy/zap/extension/ascanrulesBeta/BackupFileDisclosureUnitTest.java

```diff
@@ -43,10 +46,73 @@
             + "<p>You don't have permission to access " + PATH_TOKEN + "\n"
             + "on this server.</p>\n"
             + "</body></html>";
+    private static final String URL = "/index.html";
```

Comment:
Having an extra "directory" (e.g. `/dir/index.html`) increases the coverage and adds some more requests (still under the limits).